src/components/handlers/scheduleHandler.js

```diff
@@ -56,31 +49,25 @@ export const getLessonType = (event, currentWeek) => {
  * @param {string} targetDay -        The target day for which lessons information is required.
  *
  * @param {boolean} shouldShiftWeek - Flag defining today is Sunday and the user has clicked the 'Schedule for tomorrow' button
- *
- * @returns {string} - 		          A formatted string containing lesson information for the specified day.
+ * @param {unknown} scheduleData
  */
-export const getLessonsInfo = (targetDay, shouldShiftWeek) => {
-	// Find information about the lessons for the specified day
-	const lessonToday = dataBase.find(({ day, events }) => day === targetDay && events)
-	// If no lesson information is found for the specified day, return an error message
-	if (!lessonToday) return MESSAGE
-	// Get the current day of the week
-	const getNumberWeek = getWeek(new Date())
-	const currentWeek = shouldShiftWeek ? getNumberWeek + 1 : getNumberWeek
-	// Format lesson information for the specified day
+// eslint-disable-next-line max-len
+export const getLessonsInfo = (targetDay, shouldShiftWeek = false, scheduleData = dataBase, weekNumber = getWeekNumber(new Date())) => {
```

Comment:
I made this change at least to be able to test this function. In your implementation, you had a lot of data that came to function from the outer scopes (`dataBase` access) or was defined by the environment ( `new Date()` call), which made it impure and hard to test.